src/Api.php

```diff
@@ -4,9 +4,10 @@
 
 use Illuminate\Contracts\Container\Container;
 use Telegram\Bot\Commands\CommandBus;
+use Telegram\Bot\Events\EmitsEvents;
+use Telegram\Bot\Events\UpdateWasReceivedEvent;
```

Comment:
It could just say `UpdateWasReceived`, No need to affix with `Event`. Since the namespace itself is different and it already gives an idea of it being an event.